components/_global-footer.scss

```diff
@@ -88,11 +88,28 @@
 	}
 
 	&__bottom-bar-row {
+		margin: 0 auto;
 		padding: 16px 0;
 
 		&.wds-has-border-top {
 			border-top: 1px solid $color-slate-gray;
 		}
+
+		&.wds-has-no-padding {
+			padding: 0;
+		}
+	}
+
+	// @todo To be moved to a separate component
+	&__button-link {
+		display: inline-block;
+		font-weight: bold;
+		letter-spacing: .3px;
```

Comment:
that should be defined in global styles, should it?
why do we define it only for button link?